lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -1055,12 +1055,15 @@ void OpenCLFunction::execute(Context &ctx) {
       setKernelArg(kernel, numArgs + 6, destGradDim);
       setKernelArg(kernel, numArgs + 7, filterGradDim);
       // Zero memory for the output buffers.
-      fillBuffer(deviceBuffer_, tensors_[srcGrad], srcGrad->size(), 0,
-                 srcGrad->getElementType());
-      fillBuffer(deviceBuffer_, tensors_[filterGrad], filterGrad->size(), 0,
-                 filterGrad->getElementType());
-      fillBuffer(deviceBuffer_, tensors_[biasGrad], biasGrad->size(), 0,
-                 biasGrad->getElementType());
+      fillBuffer(deviceBuffer_,
+                 bundle_.symbolTable[std::string(srcGrad->getName())].offset,
```

Comment:
Is it really necessary to do `std::string(srcGrad->getName())`, i.e. construct a string explicitly? Even if this is the case, could you create a helper function `getValueOffset(Value *v)` which would return `bundle_.symbolTable[std::string(v->getName())].offset`?

Reply:
Currently I think it's necessary to construct the string. A helper function is a good idea though. I can do offset. And/Or I could write a helper function findSymbol(Value *v) which returns an iterator. (it->second.offset).

Reply:
Created issue #1967 to track making RuntimeBundle an object with a cleaner interface.